synapse/rest/client/v2_alpha/groups.py

```diff
@@ -25,6 +26,22 @@
 logger = logging.getLogger(__name__)
 
 
+def _validate_group_id(f):
+    """Wrapper to validate the form of the group ID.
+
+    Can be applied to any on_FOO methods that accepts a group ID as a URL parameter.
+    """
+
+    @wraps(f)
+    def wrapper(self, request, group_id, *args, **kwargs):
+        if not GroupID.is_valid(group_id):
+            raise SynapseError(400, "%s was not legal group ID" % (group_id,))
```

Comment:
```suggestion
            raise SynapseError(400, "%s is not a legal group ID" % (group_id,))
```

I realise I'm changing an old error message here, but this seems more correct?